Locales/examples/Dedekind1stNumber.g

```diff
@@ -5,9 +5,9 @@
 #! This is the free distributive lattice generated by a discrete category with one object.
 
 #! @Example
-LoadPackage( "Locales", false );
+LoadPackage( "Locales" );
```

Comment:
If you keep the `false`, the corresponding `tst` file can be executed directly without errors.

Reply:
I see, will fix it.